docs/welcome.md

```diff
@@ -15,17 +15,13 @@ ms.assetid: cb788dcf-2120-467f-9c34-c02a90e1f68f
 Welcome to .NET
 ===============
 
-> Check out the ["Getting Started with .NET Core" tutorials](core/getting-started.md) to learn how to create a simple .NET Core application. It only takes a few minutes to get your first app up and running.
+See ["Getting Started with .NET Core"](core/getting-started.md) to learn how to create a simple .NET Core app. 
 
-Welcome to .NET! You can build any kind of application you want with .NET, from cloud to IoT to games. You can start building your next application today, for Windows, Linux, Android, macOS, and iOS. There are millions of developers that use .NET to power mission critical applications, personal apps and immersive games. You can write the next one.
-
-You can develop apps and games on Windows, macOS and Linux. There are free tools that you can use to build apps or games and deploy them on servers or desktops or publish them to app stores. It's accessible to students and  used by large businesses throughout the world.
+You can build many types of apps with .NET Core, such as cloud, IoT and games. Your app can run on Windows, Linux, Android, macOS, and iOS. There are free tools that you can use to build apps or games. You can deploy them on servers or desktops or publish them to app stores. It's accessible to students.
```

Comment:
These are just my suggestions. I won't have time to respond to feedback but I recommend you get an editor to review this article. I don't think you can build micro embedded apps in .Net. It's not a universal app builder.

Too much repeated information, dead wood and market speak.